doc/tcti-spidev.md

```diff
@@ -0,0 +1,48 @@
+# SPI TCTI LTT2GO
```

Comment:
The filename `tcti-spidev.md` is a little misleading since this is about `tcti-spi-ltt2go`.

Reply:
I did not update this. But IMHO there is almost no value to this file anyways.
I did add a manpage now.